syntax = "proto3";

message PingResponse {
  string node_ip = 1;
  uint32 node_port = 2;
  string node_id = 3;
}